zammad/templates/service.yaml

```diff
@@ -13,3 +13,4 @@ spec:
       name: http
```

Comment:
i would call the file service-nginx to be in par with the other services

Reply:
Renamed the service to `zammad-nginx`.